src/ui/save-html.js

```diff
@@ -63,38 +63,46 @@ const downloadLinks = [
   },
 ];
 
-function toDownloadLink(details) {
+/**
+ * @param {typeof downloadLinks[0]} details
+ */
+function toDownloadLink(details, conf) {
   const { id, href, fileName, title, type } = details;
+  const filename = conf.shortName
+    ? `${conf.shortName}-${ISODate.format(conf.publishDate || new Date())}`
```

Comment:
Maybe format the date as a W3C date? Or does it come out nice? (Sorry not at computer, but can check tomorrow)

Reply:
I looks like: `{shortName}-2021-10-05`, which I think is a good file name.

Reply:
nah, that's good enough 👍